src/js/visual/configurator-csr.js

```diff
@@ -1771,10 +1779,12 @@ ripe.ConfiguratorCsr.prototype._registerHandlers = function() {
  * @ignore
  */
 ripe.ConfiguratorCsr.prototype._registerInitialsHandlers = function() {
-    this.owner.bind("initials", (initials, engraving, params) =>
+    // unregister previous config handlers to avoid duplicate handlers
+    this._unregisterInitialsHandlers();
+    this._onInitialsEventCallback = this.owner.bind("initials", (initials, engraving, params) =>
         this._onInitialsEvent(this, initials, engraving, params)
     );
-    this.owner.bind("initials_extra", (initialsExtra, params) =>
+    this._onInitialsExtraEventCallback = this.owner.bind("initials_extra", (initialsExtra, params) =>

```

Comment:
Make the name shorter, like `_onInitialsExtraBind`, and add the variable declaration to here: https://github.com/ripe-tech/ripe-sdk/blob/b327d752b9578f542abace9612f4276d7dc6fc27/src/js/visual/configurator-csr.js#L95

I would also take the opportunity to to shorten the name from `_onInitialsExtraEvent´ to `_onInitialsExtra` to match the `_onPostConfig` (or the other way around)